frontend/src/api/Payment.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import {BASE_URL} from "./Commons";
+
+const URL = BASE_URL + "/payment";
+
+
+export const getPaymentIntent = async (request) => {
+  const fetchData = await axios.post(URL + "/create-payment-intent", request, {
+    headers: {
+      "Content-Type": "application/json"
+    }
```

Comment:
Isn't the content type json by default? If not, this should be set for all requests in one place.